release.config.cjs

```diff
@@ -0,0 +1,44 @@
+const transform = require('./release.utils.cjs');
+/**
+ * @type {import('semantic-release').GlobalConfig}
+ */
+module.exports = {
+    branches: ['master'],
+    plugins: [
+        [
+            '@semantic-release/commit-analyzer',
+            {
+                releaseRules: [
+                    { type: 'feat', release: 'minor' },
+                    { type: 'fix', release: 'patch' },
+                    { type: 'chore', release: 'patch' },
+                    { type: 'build', release: 'patch' },
+                    { type: 'ci', release: 'patch' },
+                    { type: 'docs', release: 'patch' },
+                    { type: 'style', release: 'patch' },
+                    { type: 'perf', release: 'patch' },
+                    { type: 'test', release: 'patch' },
+                    { type: 'refactor', release: 'patch' },
+                ],
+                parserOpts: {
+                    noteKeywords: ['BREAKING', 'BREAKING CHANGE', 'BREAKING CHANGES'],
+                },
+            },
+        ],
+        [
+            '@semantic-release/release-notes-generator',
+            {
+                parserOpts: {
+                    mergePattern: /^Merge pull request #(\d+) from (.*)$/,
+                    mergeCorrespondence: ['id', 'source'],
+                },
+                writerOpts: {
+                    transform: transform,
+                },
+            },
+        ],
+        '@semantic-release/changelog',
+        '@semantic-release/npm',
+        '@semantic-release/github',
```

Comment:
you need to add this to your package.json devDependencies as well

Reply:
Yes @shayan-deriv, any `dependencies` that don't come bundled with `semantic-release` need to be added to the `package.json`